protoc_plugin/test/protos/import_public.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto2";
+
+import public "foo.proto";
+
+message A {
```

Comment:
Maybe a quick comment here or in the test file linking to the import public syntax or what it means to have a public import? https://developers.google.com/protocol-buffers/docs/proto#importing-definitions

Reply:
Good idea. Done